### PR TITLE
Change legend layout

### PR DESCRIPTION
**Pull request by Jesus89**
Related to: https://github.com/CartoDB/cartodb/issues/12619, https://github.com/CartoDB/cartodb.js/issues/1639

- Changed icon/color position in legend
- Resized legends icon to 12x12px

selection_003

### CODE REVIEW

**Comment by xavijam** [comment]
The preview is displaying last changes? Is that 12x12?

**Comment by Jesus89** [comment]
Yes 12x12

selection_005

**Comment by Jesus89** [comment]
Done!. Moving to Code Review

selection_009

**Comment by xavijam** [comment]
:+1: 🇪🇸

**Comment by Jesus89** [comment]
Added min-width to circles:

selection_010
selection_011

File: themes/scss/map/_overlays.scss [hunk 1]
```diff
@@ -464,11 +466,12 @@ $maxLegendContainerHeight: 300px;
 .Legend-categoryIcon {
   display: block;
   position: relative;
-  width: 10px;
-  height: 10px;
+  width: 12px;
```
Comment (xavijam) [inline themes/scss/map/_overlays.scss]: Shouldn't we add the min-width here?

Comment (Jesus89) [inline themes/scss/map/_overlays.scss]: I think so

**Pull request merged**